README.md

```diff
@@ -43,3 +43,21 @@ By participating in this project, you agree to abide by our [code of conduct](./
 - event naming convention: hyphenated lower case.
 	- we don't need to namespace it to individual items because it's such a small list of events. just keep it simple.
 
+## Pubsub events
+
+Events that flow through the global event broker (`app.events`, invoked as
+`this.events` by individual modules):
+
+#### `printable-key`
+
+When the user types a printable key in the urlbar, the complete string in
+the urlbar is fired as the body of the event.
+
+```
+{
+  query: 'the mar'
+}
```

Comment:
Should we also send over the full text of what the urlbar attempts to complete with? e.g. "facebook.com" here when pressing "f":

<img width="718" alt="screen shot 2016-03-02 at 10 36 18 am" src="https://cloud.githubusercontent.com/assets/23885/13468998/a6681f3e-e062-11e5-8551-d0bb473587e0.png">